core/packages/contracts/contracts/utils/MessageProtocol.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+library MessageProtocol {
```

Comment:
I'll use this library after this PR is merged.

Reply:
I don't think we need a common `MessageProtocol` library, since each "app" will have its own specific message and action types. So for example, we could just move `Action` and `Message` to `NativeTokens`.

Reply:
Ok so will we duplicate `Action` and `Message` types in each contract that implements `handle`?

Reply:
yup 👍